### ISSUE TITLE
Label of dropdown is not announced

### ISSUE THREAD

**Issue by ssaniel** [infor-design/enterprise#5127]
**Describe the bug**
Label of dropdown not announced. I'm not sure yet but maybe it is caused by adding aria-label please see attached screenshot

**To Reproduce**
1) Go to any form 
2) Press 'tab' to focus a dropdown component
![label_not_announced](https://user-images.githubusercontent.com/78124324/115837162-db607900-a44a-11eb-8d56-0f8ffea2c220.png)

**Expected behavior**
When dropdown component is focused, screenreader should announce the label of dropdown

**Version**
- ids-enterprise: 4.50.2

**Platform**
 - Infor Application/Team Name: HCM
 - OS Version: Windows 10
 - Browser Name: Chrome
 - Browser Version: Version 90.0.4430.85 (Official Build) (64-bit)

**Additional context**
Working in v4.37.3

**Comment by tmcconechy** [comment]
@ssaniel It seems ok on https://master-enterprise.demo.design.infor.com/components/dropdown/example-index.html

The code that is putting in the label for some reason must be going wrong on your end. Maybe you can debug whats happening here https://github.com/infor-design/enterprise/blob/main/src/components/dropdown/dropdown.js#L493

**Comment by ssaniel** [comment]
Thanks @tmcconechy . will check this one.
